libs/components/help-inline/testing/src/help-inline/help-inline-harness.ts

```diff
@@ -0,0 +1,106 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { TemplateRef } from '@angular/core';
+import { SkyComponentHarness } from '@skyux/core/testing';
+import {
+  SkyPopoverContentHarness,
+  SkyPopoverHarness,
+} from '@skyux/popovers/testing';
+
+import { SkyHelpInlineHarnessFilters } from './help-inline-harness.filters';
+
+/**
+ * Harness for interacting with a help inline component in tests.
+ */
+export class SkyHelpInlineHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-help-inline';
+
+  #getInlineHelpButton = this.locatorFor('.sky-help-inline');
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyHelpInlineHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkyHelpInlineHarnessFilters,
+  ): HarnessPredicate<SkyHelpInlineHarness> {
+    return SkyHelpInlineHarness.getDataSkyIdPredicate(filters);
+  }
+
+  /**
+   * Clicks the help inline button.
+   */
+  public async click(): Promise<void> {
+    await (await this.#getInlineHelpButton()).click();
+  }
+
+  /**
+   * Gets the `aria-controls` value.
+   */
+  public async getAriaControls(): Promise<string | null> {
+    return (await this.#getInlineHelpButton()).getAttribute('aria-controls');
+  }
+
+  /**
+   * Gets the `aria-expanded` value.
+   */
+  public async getAriaExpanded(): Promise<boolean> {
+    if ((await this.getAriaControls()) === null) {
+      throw new Error('aria-expanded is only set when `ariaControls` is set.');
+    }
+
+    return (await (
+      await this.#getInlineHelpButton()
+    ).getAttribute('aria-expanded')) === 'true'
+      ? true
```

Comment:
You don't need to return `true` or `false` since the `===` resolves to a boolean value.